README.md

```diff
@@ -14,23 +14,49 @@ $ python setup.py install
 ```
 3- Verify if the driver is installed
 ```
-$ python verify_installation.py
+$ pip install entry_point_inspector
```

Comment:
Could you come up with a procedure that does not involve `pip install`? Or at least could you add `--user` to it to avoid recommending `sudo pip install` (although it won't help CentOS/RHEL users who don't have `pip`).

Reply:
Doug pointed out this was a good tool to use to verify the installation that`s why  I added.

Reply:
Added the --user